### PR TITLE
feat(laodeai): added knowyourmeme handler

### LINKED ISSUE #82: laodeai know your meme handler
the title is self explanatory.

### PR DESCRIPTION
**Pull request by aldy505**
closes #82

### CODE REVIEW

File: src/services/laodeai/handlers/gist.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { sanitize } from '../../utils/sanitize.js';
+import { sanitize } from '../../../utils/sanitize.js';
```
Comment (elianiva) [inline src/services/laodeai/handlers/gist.js:1]: nodejs bisa dikasih alias ga sih? ngeselin juga lama lama numpuk gini

File: src/services/laodeai/index.js [hunk 2]
```diff
@@ -117,15 +119,18 @@ async function laodeai(context) {
  * @returns {Promise<{ url: string, type: 'image' | 'text', content: string } | { type: 'error' }>}
  */
 async function goThroughURLs(validSources) {
+  console.log(validSources);
```
Comment (elianiva) [inline src/services/laodeai/index.js:122]: jangan lupa

File: src/services/meme/index.js [hunk 3]
```diff
@@ -34,6 +34,12 @@ export function register(bot, cache) {
     await context.telegram.sendPhoto(context.message.chat.id, 'https://i.ibb.co/P1Q0650/yntkts.jpg');
   });
 
+  bot.command('homework', async (context) => {
+    const bigGroup = await isBigGroup(context);
+    if (bigGroup) return;
+    await context.telegram.sendPhoto(context.message.chat.id, 'https://i.ibb.co/541knqp/photo-2021-08-21-02-54-24.jpg');
```
Comment (elianiva) [inline src/services/meme/index.js:40]: WKWKWKWK

**Review by elianiva — APPROVED** [review]

File: src/services/laodeai/handlers/gist.js (hunk 1 above)
Comment (aldy505) [reply src/services/laodeai/handlers/gist.js:1]: bisa, pake jsconfig.json (https://code.visualstudio.com/docs/languages/jsconfig) tapi setauku kalo mau kayak gitu harus di build dulu, pake webpack atau sejenisnya. which I think a bit too much for the project.

File: src/services/laodeai/index.js (hunk 2 above)
Comment (aldy505) [reply src/services/laodeai/index.js:122]: hmm perasaan udah diapus

File: src/services/laodeai/handlers/gist.js (hunk 1 above)
Comment (elianiva) [reply src/services/laodeai/handlers/gist.js:1]: yowes lah, gini aja

Comment (aldy505) [reply src/services/laodeai/handlers/gist.js:1]: after some googling, this might work https://nodejs.org/api/packages.html#packages_subpath_imports

**Pull request merged**